src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,289 @@
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("poll_timer")}</span>
+                <span slot="description"
+                  >${this.localStr("poll_timer_long")}</span
+                >
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("poll_timer")}
+                  name="poll_timer"
+                  type="number"
+                  .value=${this._poll_timer}
+                  @value-changed=${this._handleChange}
+                ></paper-input>
+              </ha-settings-row>
+            </div>
+            <div class="card-actions">
+              <mwc-button @click=${this._publish}>
+                ${this.localStr("publish")}
+              </mwc-button>
+            </div>
+          </ha-card>
+        </div>
+      </ha-app-layout>
+    `;
+  }
+
+  protected async firstUpdated() {
+    const configEntryId = this._getConfigEntry();
+    if (!configEntryId) return;
+    const response = await this.hass.callWS({
+      type: "dynalite/get_entry",
+      entry_id: configEntryId,
+    });
+    this._entryData = (response as any).data;
+    this._name = this._entryData.name;
+    this._host = this._entryData.host;
+    this._port = this._entryData.port;
+    if (this._entryData.default) {
+      this._fade = this._entryData.default.fade;
+    }
+    const activeMap = {
+      on: 0,
+      true: 0,
+      init: 1,
+      false: 2,
+      off: 2,
+    };
+    this._activeIndex = activeMap[this._entryData.active];
+    this._active = this._activeOptions[this._activeIndex];
+    this._auto_discover = this._entryData.autodiscover;
+    this._poll_timer = this._entryData.polltimer;
+  }
+
+  private localStr(item) {
+    return this.hass.localize("ui.panel.config.dynalite." + item);
+  }
+
+  private _getConfigEntry() {
+    const searchParams = new URLSearchParams(window.location.search);
+    if (!searchParams.has("config_entry")) {
+      return false;
+    }
+    return searchParams.get("config_entry") as string;
+  }
+
+  private _handleChange(ev: PolymerChangedEvent<string>) {
+    const target = ev.currentTarget as PaperInputElement;
+    this[`_${target.name}`] = target.value;
+  }
+
+  private _handleActiveSelection(ev: CustomEvent) {
+    this._active = ev.detail.item.active_config;
+  }
+
+  private _handleAutoDiscoverChange(ev: CustomEvent) {
+    this._auto_discover = (ev.currentTarget as any).checked;
+  }
+
+  private async _publish(): Promise<void> {
+    if (!this.hass) {
+      return;
+    }
+    this._entryData.name = this._name;
+    this._entryData.host = this._host;
+    this._entryData.port = this._port;
+    this._entryData.default.fade = this._fade;
+    this._entryData.active = this._active;
+    this._entryData.autodiscover = this._auto_discover;
+    this._entryData.polltimer = this._poll_timer;
+    const configEntryId = this._getConfigEntry();
```

Comment:
Store it instead of calculating it all the time

Reply:
done